passes/opt/opt_merge.cc

```diff
@@ -51,7 +51,7 @@ struct OptMergeWorker
 
 		hashlib::commutative_hash comm;
 		for (int i = 0; i < s_width; i++)
-			comm.eat(hash_ops<std::pair<SigBit, SigSpec>>::hash({sig_s[i], sig_b.extract(i*width, width)}));
+			comm.eat_pair(sig_s[i], sig_b.extract(i*width, width));
```

Comment:
If this `hash_ops<pair<...>>::hash` pattern repeated you could simplify it with a `using` or deduplicate it with a lambda, but it's used twice with pairs of different things